src/auto-evo/simulation/food_source/EnvironmentalFoodSource.cs

```diff
@@ -42,5 +44,55 @@ public override float TotalEnergyAvailable()
         {
             return totalEnvironmentalEnergySource;
         }
+
+        protected override float StorageScore(MicrobeSpecies species, Compound compound, Patch patch,
+            SimulationCache simulationCache)
+        {
+            if (simulationCache.DayNightCycleEnabled)
+            {
+                var dayNightCycleConfiguration = SimulationParameters.Instance.GetDayNightCycleConfiguration();
```

Comment:
I *think* it is wrong to access the global simulation parameters here, auto-evo needs to only use the parameters it is given through the top level methods and everything else needs to be passed down to where it is used.